spec/integration/inventory_spec.rb

```diff
@@ -3,569 +3,611 @@
+              end
+            end
+            WebMock.disable_net_connect!
+            VCR.turn_on!
+          end
+        end
 
-    it 'Should list types with feed' do
-      types = @client.list_resource_types(feed_id)
+        let(:cassette_name) do |example|
+          description = example.description
+          description
+        end
 
-      expect(types.size).to be >= 18
-    end
+        let(:feed_id) do
+          @state[:feed_uuid]
+        end
 
-    it 'Should list types with bad feed' do
-      type = 'does not exist'
-      types = @client.list_resource_types(type)
-      expect(type).to eq('does not exist')
+        let(:wildfly_type) do
+          Hawkular::Inventory::CanonicalPath.new(feed_id: feed_id, resource_type_id: hawk_escape_id('WildFly Server'))
+        end
 
-      expect(types.size).to be(0)
-    end
+        around(:each) do |example|
+          major, minor, = @client.version
+          record("Inventory/#{secure_context}/inventory_#{major}_#{minor}", @state, cassette_name, example: example)
+        end
 
-    it 'Should list WildFlys' do
-      resources = @client.list_resources_for_type(wildfly_type.to_s)
+        it 'Should list feeds' do
+          feeds = @client.list_feeds
 
-      expect(resources.size).to be(1)
-    end
+          expect(feeds.size).to be > 0
+        end
 
-    it 'Should list WildFlys with props' do
-      resources = @client.list_resources_for_type(wildfly_type.to_s, fetch_properties: true)
+        it 'Should list resources for feed' do
+          resources = @client.list_resources_for_feed feed_id
 
-      expect(resources.size).to be(1)
-      wf = resources.first
-      expect(wf.properties['Hostname']).not_to be_empty
-    end
+          expect(resources.size).to be(2)
+        end
 
-    it 'Should List datasources with no props' do
-      type_path = CanonicalPath.new(feed_id: feed_id, resource_type_id: hawk_escape_id('Datasource'))
-      resources = @client.list_resources_for_type(type_path.to_s, fetch_properties: true)
+        it 'Should list all the resource types' do
+          types = @client.list_resource_types
+          # new API returns only the feedless types here, while the old one returned all the types
+          expect(types.size).to be > 0
+        end
 
-      expect(resources.size).to be > 0
-    end
+        it 'Should list types with feed' do
+          types = @client.list_resource_types(feed_id)
 
-    it 'Should list URLs' do
-      type_path = CanonicalPath.new(resource_type_id: hawk_escape_id('URL'))
-      resources = @client.list_resources_for_type(type_path.to_s)
+          expect(types.size).to be >= 18
+        end
 
-      expect(resources.size).to be(1)
-      resource = resources[0]
-      expect(resource.instance_of? Hawkular::Inventory::Resource).to be_truthy
-      # depends how pinger is fast
-      expect(2..6).to cover(resource.properties.size)
-      expect(resource.properties['url']).to eq(URL_RESOURCE)
-    end
```

Comment:
This should now work after https://github.com/hawkular/hawkular-services/pull/51 was merged. But I am ok with omitting it from the specs, because we don't use it anymore. @pilhuhn wdyt? do we want to still test that after url is correctly added and possibly if the pinger pings it? Is a spec in ruby gem the right place to do that?